### ISSUE TITLE
Please readd /bin/sh to hadolint docker image

### ISSUE THREAD

**Issue by mrueg** [hadolint/hadolint#218]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
/bin/sh available on hadolint docker image
### Actual behavior
None found
### Steps to reproduce the behavior
Since this change, there's no /bin/sh in the hadolint docker image:
https://github.com/hadolint/hadolint/commit/8c2642232cbc4daa27f553bc3b4f1ddb63ba4b70

I added hadolint to my CI system and which executes steps within a shell, that make /bin/sh necessary. Before building my own images, I'd like to ask if you could you consider readding /bin/sh and maybe publish the other one as hadolint-minimal?

**Comment by lorenzo** [comment]
what ci system do you use?

**Comment by mrueg** [comment]
I used drone CI (https://github.com/drone/drone). My CI config looks similar to this:
```
lint:
    image: hadolint/hadolint
    group: lint
    pull: true
    commands:
      - hadolint --version
      - hadolint Dockerfile
```

**Comment by lorenzo** [comment]
@mrueg would you like to help finish this PR? we are only missing a few changes before we can merge it and offer an official docker image with a shell https://github.com/hadolint/hadolint/pull/210

**Comment by wevcode** [comment]
@lorenzo I am using GitLab CI with GitLab Runner (Docker Mode) which also expects /bin/sh.

**Comment by lorenzo** [comment]
@wevcode are the instructions for gitlab here any help? https://github.com/hadolint/hadolint/blob/master/docs/INTEGRATION.md#gitlab-ci

**Comment by wevcode** [comment]
@lorenzo This exampe uses Docker-In-Docker which will force you to use a privileged docker runner. (https://docs.gitlab.com/ee/ci/docker/using_docker_build.html)

Hadolint does not need direct docker daemon access so it should be able to run without privileged mode. Because GitLab CI performs some tasks before starting the actual job, `/bin/sh` is needed. AFAIK `busybox` should be enough.

**Comment by wevcode** [comment]
@lorenzo This code snippet will create more confusion because it will throw another error if you are not using such a privileged runner.

**Comment by lorenzo** [comment]
@wevcode would you like to help finish #210 ? the original author got very close to an acceptable place, but decided not to follow through.

**Comment by wevcode** [comment]
@lorenzo Seems simple. Should I open a new PR?

**Comment by lorenzo** [comment]
@wevcode yes please

**Comment by lorenzo** [comment]
The `hadolint/hadolint:latest-debian` image is now available
